plugins/backstage-plugin-devcontainers-react/README.md

```diff
@@ -14,11 +14,15 @@ _Note: While this plugin can be used standalone, it has been designed to be a fr
 
 ### Standalone features
 
-- Custom hooks for reading your special Dev Container metadata tag inside your repo entities, and providing ready-made links to opening that repo in VS Code
+- Custom hooks for reading your special Dev Container metadata tag and VS Code
+  launch URI inside your repo entities, and exposing that URI for opening the
+  repo in VS Code
```

Comment:
I think hard wrapping makes sense for anything that can contain code in it.

Files in a repository especially probably tend to get viewed in a variety of terminals and editors, which may not all have support for soft wrapping at an arbitrary column.

Also the diff for long lines can be kinda gnarly since it often wraps at the window edge (GitHub at least does this, not sure if there is configuration for it though).

Whatever we decide we should standardize for sure.  The top-level readme has some of both as well.

coder/coder does not seem to use hard wrapping, so might make sense to follow even if it makes me a little grumpy.  :laughing:

Reply:
Unwrapped for now!